interface/main/tabs/js/frame_proxies.js

```diff
@@ -37,6 +37,12 @@ left_nav.setPatient = function(pname, pid, pubpid, frname, str_dob)
     var new_patient=new patient_data_view_model(pname,pid,pubpid,str_dob);
     app_view_model.application_data.patient(new_patient);
     app_view_model.application_data.therapy_group(null);
+
+    if (WindowTitleAddPatient)
+    {
+        top.document.title = WindowTitleBase + " - " + pname;
+    }
```

Comment:
also need to do the reverse when closing the patient (in cases where close the patient without opening a new patient). Check out the clearPatient() function in interface/main/tabs/js/tabs_view_model.js for place for doing that.